style/styles.css

```diff
@@ -376,3 +397,107 @@ header {
 #get-in-touch {
   margin-top: 6px;
 }
+
+/* Desktop version min-width of 768px) */
+@media only screen and (min-width: 768px) {
+  .navbar ul {
+    display: flex;
+    list-style: none;
+  }
+
+  .navbar ul li {
+    margin: 10px;
+  }
+
+  .navbar ul li a {
+    text-decoration: none;
+    color: #172b4d;
+  }
+
+  .burgerIcon {
+    display: none;
+  }
+
+  .headLine {
+    background: white url("../images/header-bg-desktop.svg") no-repeat;
+    padding-left: 260px;
+    padding-right: 260px;
+    padding-top: 100px;
+  }
+
+  .logo {
+    padding-left: 155px;
+  }
+
+  .grid-container {
+    padding-left: 142px;
+    padding-right: 142px;
+  }
+
+  .grid-item {
+    display: flex;
+    flex-direction: row;
+    gap: 46px;
+  }
+
+  .profile-image-mobile {
+    display: none !important;
+  }
+
+  .profile-image-desktop {
+    display: block !important;
+  }
+
+  .about-my-self {
+    display: flex;
+    flex-direction: row;
+    gap: 16px;
+  }
```

Comment:
Please review the Figma template and make sure your design matches it. You should fix margins, padding, widths, and heights among other things.